calibrators/intrinsic_camera_calibrator/intrinsic_camera_calibrator/intrinsic_camera_calibrator/camera_calibrator.py

```diff
@@ -795,6 +874,23 @@ def process_evaluation_results(
     def on_consumed(self):
         self.data_source.consumed()
 
+    def save_parameters(self, filename):
+        data_coll_params = self.data_collector.get_parameters_values()
+        board_params = self.board_parameters.get_parameters_values()
+        detector_params = self.detector.get_parameters_values()
+        calibrator_type = self.calibrator_type_combobox.currentData()
+        print(calibrator_type.value["name"], flush=True)
```

Comment:
delete or use logging

Reply:
thanks, i will fix it

Reply:
f2ad5cce4cffc72284d0b575020c40851475872e

Reply:
This is not the commit that addressed the comment...